src/pages/FrontPage.scss

```diff
@@ -0,0 +1,83 @@
+@import '../assets/css/fonts';
+
+.content {
+  margin-top: 75px;
+}
+
+.column {
+  display: inline-block;
+  vertical-align: top;
+}
+
+.leftColumn {
+  composes: column;
+  width: 25%;
+}
+
+.rightColumn {
```

Comment:
Ok, takk @arve0, tror det kom gjennom til slutt ;)
Ja, det høres fornuftig ut. Da gjør vi det sånn at med mindre det skulle dukke opp en god grunn, så merger vi med squash. Jeg har latt begge mulighetene være tillatt, kan dere komme på en god grunn for å merge uten squash i blant? Hvis ikke kan jeg skru av vanlig merge sånn at det bare er lov å squashe på merge.

Takk for tålmodigheten, folkens! :)